gke-nfs-lb/charts/v0.0.1/nfs-csi-lb/templates/csi-nfs-controller-lb.yaml

```diff
@@ -0,0 +1,120 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-nfs-controller
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-nfs-controller
+  template:
+    metadata:
+      labels:
+        app: csi-nfs-controller
+    spec:
+      hostNetwork: true  # controller also needs to mount nfs to create dir
```

Comment:
this is not needed since we don't have the provisioner right?

Reply:
good point. done